build/cicd/sonarqube/entrypoint.sh

```diff
@@ -91,6 +93,9 @@ case ${LANGUAGE} in
         ;;
 esac
 
+# Print debug log without sensitive info
+debugParams=$(echo "${params/$TOKEN/**********}")
+printf "\n------------- [DEBUG] -------------\nsonar-scanner %s -X\n\n" "$debugParams"
```

Comment:
I'd prefer to make the log simper, just print what you want directly.

`echo "sonar-scanner ${params/$TOKEN/**********} -X"`